lib/grapevine_web/live/post_live.ex

```diff
@@ -85,13 +88,17 @@ defmodule GrapevineWeb.PostLive do
   def handle_info({:updated_post, post}, socket) do
     p_index = Enum.find_index(socket.assigns.posts, fn x -> x.id == post.id end)
     posts = List.replace_at(socket.assigns.posts, p_index, post)
-
     {:noreply, assign(socket, posts: posts)}
   end
 
+  def handle_info({:category_filter, "0" }, %{assigns: %{all_posts: all_posts}} = socket) do
+    #push_patch(socket, to: Routes.live_path(socket, MyLive, page + 1))
+    {:noreply, assign(socket, posts: all_posts,category_id: "0",)}
```

Comment:
Nit, but.

```suggestion
    {:noreply, assign(socket, posts: all_posts,category_id: "0",)}
```

Nice PR, @neal-bpm!